Observatory-Control-System/BLOCKS/block-jira-project-workflow.rst

```diff
@@ -0,0 +1,245 @@
+
+*Transition to Ready*: Once all of the steps above have been completed, the Assignee is responsible for transitioning the ticket to Ready. 
+
+*Transition to Proposed*: After initial assignee assessment if the work cannot be completed by the assignee in a timely manner in accordance with the observing task priority, it can be returned to Proposed for re-assignment. 
+
+**Ready**
+^^^^^^^^^
+This state is reserved for observing tasks which have been prepared, tested, reviewed, and deployed at the summit and are ready for execution.
+
+While in the Ready state, a task can be added to the plan at any time. 
+Tasks will be planned for observation in accordance with the priority they were assigned in the Proposed state. 
+A review of the tasks which are in the Ready state is to be carried out as part of the regular Commissioning Science planning meetings. 
+
+The steps expected to be completed in this state include:
+
+* Planning of the task for execution by the Commissioning Science team. 
+* Execution of the task at the Summit.
+* Review of the data and assessment of task completion by the Proposer and Assignee. 
+
+*Transition to Preparation*: If an issue is found with the observing task after it has been transitioned to Ready, it may be transitioned back to Preparation.
+This transition could be triggered by either the Assignee, the Proposer, or the user responsible for executing the task on the Summit. 
+
+*Transition to Done*: If the task has been executed and the data collected deemed satisfactory by the Proposer and there is no need to repeat the test, 
+it can be transitioned to *Done* and removed from further planning consideration. 
+
+End states
+----------
+
+The end states are the ultimate destination of a reported ticket.
+When a ticket reaches one of these states, is will no longer be considered for scheduling.
+We start by explaining these states because they are referenced throughout the remainder of the document, specifically when describing state transitions.
+
+**Done**
+^^^^^^^^
+All work required to complete the observing block has been Done. There is no current plan to repeat the task. 
+
+**Invalid**
+^^^^^^^^^^^
+The requested observing block is no longer valid, was filed by mistake, or has been superseded by another block or other events.
+
+There are multiple ways that tickets reach these states, which are explained in the sections below.
+The Invalid state is special because it can be transitioned to from any active state in the workflow.
+
+Both Invalid and Done allow transition out of these states, which is to return to the Proposed state and start the workflow over again.
+This transition should be rare. 
+It is reserved for cases when a job is erroneously transitioned to a completed state, or a later analysis of the observing task deems it was not satisfactorily completed. 
+The ticket is then transitioned to Proposed so it can be prepared and prioritized accordingly by a Commissioning Science Team.
+
+
```

Comment:
Here we could add one more section providing a kind of a "checklist" for both roles, to make sure they do not forget anything. I prepared something, in case you think it is valuable (feel free to reject it, of course):

```
BLOCK Ticket Lifecycle Responsibilities Checklist
=================================================

Proposer Checklist
------------------

Initial Ticket Submission
^^^^^^^^^^^^^^^^^^^^^^^^^

- Identify the need for an observing task.
- Draft a clear description of the observing task, including high-level objectives and specific requirements.
- Gather and attach supporting materials such as relevant tickets (SITCOM, LVV) or documentation on Confluence pages.
- Create and submit the BLOCK Jira ticket.
- Provide an original time estimation for the task execution, including preparation and any additional time for configuration or logging.

Collaboration during Preparation
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

- Review and discuss the preparation details with the Assignee to ensure all task requirements are clearly understood.
- Provide any additional information or clarifications needed by the Assignee.

Final Verification and Sign-Off
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

- Review the outcomes of the executed task to ensure all objectives have been met.
- Verify the completeness and accuracy of the data or results collected.
- Provide final sign-off and transition the ticket to "Done" if satisfied with the task completion.
- Initiate reevaluation if the task outcomes do not meet the expected criteria, by transitioning the ticket back to "Proposed."

Assignee Checklist
------------------

Task Preparation and Planning
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

- Receive and review the BLOCK ticket once assigned by the Commissioning Science team.
- Collaborate with the Proposer to fully understand the task requirements and expectations.
- Plan the detailed preparation of the task, including method selection (e.g., JSON BLOCK, SAL Script) and setup.
- Prepare and configure all necessary components or settings for the task.

Marking as Ready
^^^^^^^^^^^^^^^^

- Conduct a thorough final check to ensure all preparations meet the required standards and are fully operational.
- Validate all configurations and preconditions for the task execution.
- Mark the BLOCK ticket as "Ready" indicating readiness for execution.

Advocacy and Tracking
^^^^^^^^^^^^^^^^^^^^

- Advocate for the task in planning meetings to ensure it is scheduled based on its priority.
- Track the status of the task and update its priority if necessary.
- Ensure the task is added to the execution plan and adequately prepared for execution at the summit.

Role during Execution
^^^^^^^^^^^^^^^^^^^^^

- Provide necessary instructions or clarifications to the Observing Specialist or execution team at the summit.
- Monitor the execution of the task and be available to address any issues that may arise during the execution phase.

Observing Specialist Checklist (Execution Focused)
--------------------------------------------------

- Review the task details and any specific instructions before execution.
- Execute the observing task as planned during the scheduled night.
- Report any issues or deviations during execution to the Assignee. 
```

Reply:
I like this idea, it's kind of a different way to visualize the information that might be easier to reference. I will add it as part of my edits and we can see how it renders.